ons_alpha/bulletins/models.py

```diff
@@ -58,3 +60,13 @@ class BulletinPage(BasePage):
             ObjectList(BasePage.settings_panels, heading="Settings", classname="settings"),
         ]
     )
+
+    def get_latest_release(self):
+        parent = self.get_parent().specific
+        latest = BulletinPage.objects.live().descendant_of(parent).order_by('-release_date').first()
```

Comment:
```suggestion
        parent = self.get_parent()
        latest = BulletinPage.objects.live().child_of(parent).order_by('-release_date').first()
```

Reply:
however, this should move to `BulletinSeriesPage` so this would become:

```suggestion
        latest = BulletinPage.objects.live().child_of(self).order_by('-release_date').first()
```